help.php

```diff
@@ -12,19 +45,48 @@
                         </ol>
                         <div class="card mb-4">
                             <div class="card-body">
-                                This page is an example of using the light side navigation option. By appending the
-                                <code>.sb-sidenav-light</code>
-                                class to the
-                                <code>.sb-sidenav</code>
-                                class, the side navigation will take on a light color scheme. The
-                                <code>.sb-sidenav-dark</code>
-                                is also available for a darker option.
+                            <h1>Welcome to INA TOOL</h1>
+                                <h3>Dashboard Overview</h3>
+                                <ul>This is the central hub where you can view the istitutional statements visualization and manage your data.</ul>
+
+                                <h3>Upload your File & Template</h3>
+                                <ul>
+                                You can upload only <i>.csv</i> or <i>.txt</i> files. Please, use this <a href="Institutional_statement_template - INA.txt" download>template</a>. Alternatively, the file must respect the columns ordering and naming described below:

```

Comment:
Shall we link to the [excel template](https://nlesc.sharepoint.com/:x:/s/esi-far77/Eeuypd5ADlhFuj6bqazIt5IB4Yxy1of0iiIVM-yYt7Akvw?e=BXtoMa_) instead? It has helpful conditional formatting to indicate which fields are required. We should explicitly say to export/download as CSV for now, to make sure they don't try to upload an excel file instead.

Reply:
Yes!